-------------------------------------------------------------------------------
Test set: TestSocket
-------------------------------------------------------------------------------
Tests run: 3, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 2.651 s <<< FAILURE! - in TestSocket
calldog(TestSocket)  Time elapsed: 0.016 s  <<< FAILURE!
java.net.BindException: Address already in use: bind
	at TestSocket.calldog(TestSocket.java:24)

initCliente(TestSocket)  Time elapsed: 2.214 s  <<< FAILURE!
java.net.ConnectException: Connection refused: connect
	at TestSocket.initCliente(TestSocket.java:17)

testSocket(TestSocket)  Time elapsed: 0.001 s  <<< FAILURE!
java.net.BindException: Address already in use: bind
	at TestSocket.testSocket(TestSocket.java:13)

